app/controllers/samples_controller.rb

```diff
@@ -825,7 +840,15 @@ def update
     respond_to do |format|
       if @sample.update(sample_params)
         format.html { redirect_to @sample, notice: 'Sample was successfully updated.' }
-        format.json { render :show, status: :ok, location: @sample }
```

Comment:
Tricky find: this was directing to the `show` action for `@sample`, which implicitly called the `jbuilder` template below. I want to get rid of the implicit behavior of the jbuilder, so I wrote the code out explicitly here.

Reply:
Cool find.
These convention behaviors can be a nice help sometimes, but as you say it can be tricky to find/track or event notice.

On the other hand, we need to standardize formats that are returned by our app, and constant custom returns are hard to maintain. Anyway, that would be an issue for an API revamp.

Reply:
Got it!